lib/ansible/modules/storage/netapp/na_ontap_igroup.py

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/python
+''' this is igroup module
+
+ (c) 2018, NetApp, Inc
+ # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+'''
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+
+module: na_ontap_igroup
+short_description: ONTAP iSCSI igroup configuration
+extends_documentation_fragment:
+    - netapp.na_ontap
+version_added: '2.6'
+author: chhaya gunawat (chhayag@netapp.com), Chris Archibald (carchi@netapp.com), Suhas Bangalore Shekar (bsuhas@netapp.com)
+
+description:
+    - create, destroy or rename Igroups and add or remove initiator in igroups.
+
+options:
+  state:
+    description:
+    - Whether the specified Igroup should exist or not.
+    choices: ['present', 'absent']
+    default: present
+
+  name:
+    description:
+    - The name of the lun to manage.
+    required: true
+
+  initiator_group_type:
+    description:
+    - Type of the initiator group. Possible values "fcp", "iscsi", "mixed".
+    - Required when C(state=present).
+
+  new_name:
+    description:
+    - New name to be given to initiator group..
+
+  ostype:
+    description:
+    - OS type of the initiators within the group. The default value if not specified is "default".
+
+  initiator:
+    description:
+    - WWPN, WWPN Alias, or iSCSI name of Initiator to add or remove.
+
+  bind_portset:
+    description:
+    - Name of a current portset to bind to the newly created igroup.
+
+  force:
```

Comment:
Should be specific about what's being forced in the arg name, something like: `force_removal_with_mapped_luns`. These tend to grow over time, and generic names like `force` are problematic for someone just reading the playbook to know what it does without consulting the docs.

Reply:
Fixed